7_react/react2/week1/my-app/src/SearchResults/index.js

```diff
@@ -0,0 +1,17 @@
+import { AppContext } from "../AppContext";
+import { useContext } from "react";
+import "./index.css";
+
+const SearchResults = () => {
+  const { items } = useContext(AppContext);
+
+  const searchResults = items.map((item) => <li>{item.login}</li>);
```

Comment:
Remember to assign a key every time you use a map. This is probably showing you a warning in the browser console as well :)